src/store/index.js

```diff
@@ -85,12 +85,28 @@ const store = createStore({
               'token': state.userToken
             }
           }).then(res => {
-            commit('setPolls', res.data)
-            console.log(state.polls)
+            commit('setPolls', res.data.rows)
           })
       } catch (error) {
         console.log(error)
       }
+    },
+
+    //for adding poll
+    async addPoll({ state }, { title, options }) {
+      try {
+        await axios.post(process.env.VUE_APP_BASE_URL + process.env.VUE_APP_POLLADD_API,
```

Comment:
@PiyushInnotech can you create an axios interceptor for passing token into header